service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/SendNoticeOfHearingHandlerTest.java

```diff
@@ -144,4 +209,32 @@ void shouldSendNoticeOfHearingToRepresentativesAndNotRepresentedRespondentsByPos
         verifyNoMoreInteractions(sendDocumentService);
         verifyNoInteractions(notificationService);
     }
+
+    @Test
+    void shouldSendNoticeOfHearingToUnrepresentedOthersByPost() {
+        final RespondentParty otherRespondent = mock(RespondentParty.class);
+        DocumentReference noticeOfHearing = mock(DocumentReference.class);
+
+        given(sendDocumentService.getStandardRecipients(CASE_DATA)).willReturn(new ArrayList<>());
+        given(HEARING.getNoticeOfHearing()).willReturn(noticeOfHearing);
+        given(HEARING.getOthers()).willReturn(emptyList());
+        given(otherRecipientsInbox.getSelectedRecipientsWithNoRepresentation(emptyList())).willReturn(
+            Set.of(otherRespondent));
+
+        underTest.sendNoticeOfHearingByPost(new SendNoticeOfHearing(CASE_DATA, HEARING));
+
+        verify(sendDocumentService).sendDocuments(CASE_DATA, List.of(noticeOfHearing), List.of(otherRespondent));
+        verify(sendDocumentService).getStandardRecipients(CASE_DATA);
+
```

Comment:
others isn't returned in getStandardRecipients so is there benefit in verifying it here?